charts/oidc-proxy/templates/oidc-proxy-deployment.yaml

```diff
@@ -73,14 +73,14 @@ spec:
         {{- end }}
         volumeMounts:
           {{- if .Values.tcell.enabled }}
-          - mountPath: /etc/apache2/tcell_agent.config
+          - mountPath: /etc/httpd/tcell_agent.config
```

Comment:
httpd uses different mount paths, used https://github.com/broadinstitute/terra-helmfile/pull/3247/files as a reference

Reply:
There was another rename: `ca-bundle.crt` -> `server-ca-bundle.crt` but I don't see that referenced in TDR.